utility_analysis_new/dp_engine.py

```diff
@@ -38,8 +37,22 @@ def aggregate(self,
         self._is_public_partitions = public_partitions is not None
         result = super().aggregate(col, params, data_extractors,
                                    public_partitions)
+        error_aggregators = self._create_aggregate_error_compound_combiner(
```

Comment:
Nit: `error_combiners`

Reply:
I'd prefer to use something like `aggregate_error_combiners` since we are combining errors in general but this is specifically about the aggregate errors.